pop.css

```diff
@@ -0,0 +1,100 @@
+
+.main-for-pop-up {
+  background-color: rgba(0, 0, 255, 0.438);
```

Comment:
- [ ] Your JS syntax is correct, keep up the great job, but I think there are a few things you will need to work on the styling, this gif below compared to the image design, their pop-up container (over-lay is not meant to have that purple color, instead it should be light gray with some transparency, please refer to the Figma design for `color, padding, margin, font-family, and  font-size
- [ Your Desin]
- 
![anicet](https://user-images.githubusercontent.com/79058364/157830773-d652386a-c363-4c38-841c-0b811f60ce79.gif)
 

- [ ] To change
- your top section ( canopy . backend ...) is not aligned and does not match the design, please fix that.
- your closing button is fixed but when the user scrolls the pop-up window, as you can see from the gif, it gets confused with the text and images which is not pretty.
- In your paragraph, please select the correct color position, font size, font-weight, and font family as per the Figma design
- Your technologies, please align and style them as per the Figma design below

- [LET'S TAKE A LOOK AT THE FIGMA]
-  MOBILE (470)

<img width="285" alt="mob pop" src="https://user-images.githubusercontent.com/79058364/157832593-19d39a9f-746f-43e2-8571-626e42b634cc.png">

-  DESKTOP (992)
<img width="592" alt="pop desk" src="https://user-images.githubusercontent.com/79058364/157832674-91fab0c8-03cf-4bb8-a9ba-25e3c4b3bb29.png">

Reply:
Thank you for your feedback @BenMukebo 

You are really doing a great job 💯 

Here bellow in my next commits that I took time to respect each one of your requested changes. 
nevertheless one point stay unsolved, When confuguring the close button, it acting on my js syntax,
Actually; You will see that I've put an other one on each pop-up window and I really tried over and over again ! But I stay open to any help

Hope that you'll understand !!

Thank you again and happy coding !!

Reply:
Hello @AnicetFantomas,
 Great work, am proud of your accomplishment 😄, congrats 💯